doc/Command_Index_User.tex

```diff
@@ -1099,6 +1126,108 @@ \subsection{Distances}\label{distances}
 
 Default: $\unit[-0.05]{cm}$
 
+\macroname{nabcinterelementspace}{}{gsp-default.tex}
+Space between elements in ancient notation.
+
+Default: \unit[0.06927]{cm} plus \unit[0.00182]{cm} minus \unit[0.00363]{cm}
+
+\macroname{nabclargerspace}{}{gsp-default.tex}
+Larger space between elements in ancient notation.
+
+Default: \unit[0.10938]{cm} plus \unit[0.01822]{cm} minus \unit[0.00911]{cm}
+
+\macroname{clivisalignmentmin}{}{gsp-defualt.tex}
+When \verb=\gre@clivisalignment= is 2, this distance is the maximum length of the consonants after vowels for which the clivis will be aligned on its center.
+
+Default: \unit[0.3]{cm}
+
+\macroname{clefchangespace}{}{gsp-default.tex}
+Space around a clef change.
+
+Default: \unit[0.01768]{cm} plus \unit[0.00175]{cm} minus \unit[0.01768]{cm}
+
+
+\subsection{Penalties}\label{penalties}
+Penalties are used by \TeX\ to determine where line and page breaks should occur.  Gregorio\TeX\ modifies or defines a few of its own to help with that process in scores.
+
+\macroname{\textbackslash grebrokenpenalty}{}{gsp-default.tex}
+The vertical penalty inserted after a break on a clef change.
+
+Default: 0
+
+\macroname{\textbackslash greclubpenalty}{}{gsp-default.tex}
+The club penalty (determines how important it is to prevent orphans from occurring).
+
+Default: 0
+
+\macroname{\textbackslash grewidowpenalty}{}{gsp-default.tex}
+The widow penalty (determines how important it is to prevent widows from occurring).
+
+Default: 0
+
+\macroname{\textbackslash greemergencystretch}{}{gsp-default.tex}
+The value of the last ditch stretch for overfull boxes.
+
+Default: \verb=\emergencystretch=
+
+\macroname{\textbackslash greendafterbarpenalty}{}{gsp-default.tex}
+The end after bar penalty.
+
+Default: $-200$
+
+\macroname{\textbackslash greendafterbaraltpenalty}{}{gsp-default.tex}
+The alternate end after bar penalty (used when there is no text under the bar).
+
+Default: $-200$
+
+\macroname{\textbackslash greendofelementpenalty}{}{gsp-default.tex}
+The end of element penalty.
+
+Default: $-50$
+
+\macroname{\textbackslash greendofsyllablepenalty}{}{gsp-default.tex}
+The end of element penalty.
+
+Default: $-50$
+
+\macroname{\textbackslash greendofwordpenalty}{}{gsp-default.tex}
+The end of element penalty.
+
+Default: $-100$
+
+\macroname{\textbackslash grehyphenpenalty}{}{gsp-default.tex}
+The hyphen penalty.
+
+Default: $-200$
+
+\macroname{\textbackslash grenobreakpenalty}{}{gsp-default.tex}
+Penalty to prevent a line break.
+
+Default: 10001
+
+\macroname{\textbackslash grenewlinepenalty}{}{gsp-default.tex}
+Penalty to force a line break.
+
+Default: -10001
+
+\macroname{\textbackslash grelooseness}{}{gsp-default.tex}
+The looseness within a score.
+
+Default: \verb=\looseness=
+
+\macroname{\textbackslash gretolerance}{}{gsp-default.tex}
+The tolerance within a score.
```

Comment:
I think it would be way too complex to explain here, but I admit I don't know which website or doc it should point to... Maybe https://en.wikibooks.org/wiki/TeX/pretolerance ?